### PR TITLE
feat: Record HTTP server request parameters as `message`

### PR DESCRIPTION
**Pull request by dustinbyrne**
image

### CODE REVIEW

**Review by apotterri — CHANGES_REQUESTED** [review]
This looks good, except the tests fail for python 3.5. Once that gets sorted out, it'll be good to go.

**Review by apotterri — DISMISSED** [review]

**Review by apotterri — CHANGES_REQUESTED** [review]
Oops, there is one more thing needed here. Can you add a CHANGELOG entry, please?

File: appmap/test/test_django.py [hunk 1]
```diff
@@ -48,3 +48,85 @@ def test_http_capture(events):
         'status_code': 404,
         'mime_type': 'text/html'
     }
+
+
+def test_message_capture_get(events):
+    client = django.test.Client()
+    client.get('/test', { 'my_param': 'example' }, content_type='application/x-www-form-urlencoded')
```
Comment (dividedmind) [inline appmap/test/test_django.py:55]: `GET` requests shouldn't take `content_type` as it relates to body, which GET requests lack.

**Review by dividedmind — APPROVED** [review]
Just one small note, plus missing changelog entry.

**Comment by apotterri** [comment]
Merged here: 2a5be737d7ba8e47782f946c6cb61dd34f147a53

**Pull request closed without merging**